### PR TITLE
add draft to guidelines from slack discussion

### PR DESCRIPTION
**Pull request by lpantano**
Following slack discussion we agree on adding some text on PR review to make it more transparent to new people or as a reminder.

@netlify /docs/tutorials/contributing_to_nf-core/contributing_to_pipelines

### CODE REVIEW

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (ewels) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:88]: Indent these as they're sub-items of the above

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (ewels) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:89]: Link to docs for first release

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (ewels) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:94]: Indent as it's an explanation

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (ewels) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:96]: Squash merging -> squashing 

Maybe needs a bit more explanation and is a sub point of the merge commit one above

**Review by ewels — COMMENTED** [review]
Needs a little polishing yet I think, left a few comments. Nothing major though.

What do folks think about having this as a new dedicated _Guidelines_ page and linking to it from here?

I had envisioned it more as a set of rules / standards that we follow rather than instructions in a tutorial tbh..

**Comment by ewels** [comment]
Also maybe some mention that it's fine / encouraged for other people to push to your PR. For example fixing linting (can mention bot commands) but also to build on or fix code if appropriate.

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (jfy133) [reply sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:88]:
```suggestion
- Exception: merges to master for releases requires **two** reviews.
```

I think the grammar might confuse some people

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (jfy133) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:89]:
```suggestion
- Exception: pseudo PR reviews for a first release needs **review from core or maintainer team**.
```

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (jfy133) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:90]:
```suggestion
- If it's there is a major change in any type of PR (from modules to pipelines) then it's ok to request a second review - either the author or reviewer can do this.
```

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (jfy133) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:91]:
```suggestion
- PRs that have approving reviews but remaining open questions can be merged by the author after addressing the questions to a common-sense level of satisfaction.
```

**Review by jfy133 — COMMENTED** [review]
Couple of prhasing tweaks but overall LGTM after the indentation from Phil (agree reads a bit weird as they are explanation)

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (lpantano) [reply sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md]: sorry, I cannot find exactly this? I found pipeline release but not first release

**Comment by lpantano** [comment]
thanks! I made a bunch of changes following the comments, much better now, not sure if I am 100% in the indents of all the points, let me know. 
Agree to have it outside tutorial, are you thinking Guidelines/Pipelines?

**Comment by lpantano** [comment]
or just under Guidelines?

**Comment by lpantano** [comment]
@ewels and @jfy133 , I think this is almost there. Just need to figure out the final place for this. Is inside Guidelines a good place?

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (jfy133) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:90]:
```suggestion
  - Exception: pseudo PR reviews for a first release needs **at least one review from core or maintainer team**.
```

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (jfy133) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:91]:
```suggestion
  - If there is a major change in any type of PR (from modules to pipelines) then it's ok to request a second review - either the author or reviewer can do this.
```

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (jfy133) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:93]:
```suggestion
  - If the reviewer feels uneasy about a remaining open question then they shouldn't leave a only a comment or approval, but rather a request-changes.
```

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (jfy133) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:94]:
```suggestion
  - **request changes** reviews can be [dismissed](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/dismissing-a-pull-request-review) by the PR author if considered out of date and resolved.
  - If a PR has comments from one reviewer with open questions, and requests to re-review but does not perform the re-review, the PR may be merged if the first reviewer does not perform the review after 3 months if the PR gets an independent approval from someone else.
```

Something that was discussed in the last maintainers meeting

**Review by jfy133 — COMMENTED** [review]
Ugh sorry @lpantano ! The first re-review got buried in my notifications :see_no_evil: 

Re the location: Reading through it I think a guidelines page would make sense - you use a lot of 'SHOULD' language which would fit, but if we do move there I feel we need to tweak the language to follow e.g. the modules guidelines (with a strict definition of SHOULD/MUST/MUST NOT) etc.

Each major point in this list should be become it's own header which we can then number using that astro plugin @mashehu has added to the top of e.g. the modules guidelines, and we can 'cite' them.

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (jfy133) [inline sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md:46]: I guess this should be changed to point to theother file?

**Comment by lpantano** [comment]
@nf-core-bot fix linting

**Comment by lpantano** [comment]
ok, got it, I think links are working, fix some typos, fix links in the tutorial section. The only think missing is:
-> Link to docs for first release https://github.com/nf-core/website/pull/2780#discussion_r1793939484
I don't know where is that...

File: sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md
Comment (mashehu) [reply sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md]: https://nf-co.re/docs/tutorials/adding_a_pipeline/first_release

**Review by jfy133 — APPROVED** [review]
:tada: :tada:

**Comment by lpantano** [comment]
@jfy133 I don't think I can merge.

**Pull request merged**
